packages/dds/tree/src/test/shared-tree/editing.spec.ts

```diff
@@ -2739,64 +2749,49 @@ describe("Editing", () => {
 
 			it("sequence field node exists constraint", () => {
 				const tree = makeTreeFromJson([]);
+				const { undoStack, unsubscribe } = createTestUndoRedoStacks(tree.events);
+
 				const bPath = {
 					parent: undefined,
 					parentField: rootFieldKey,
 					parentIndex: 1,
 				};
```

Comment:
`bPath` seems like an odd name given we use this path to refer to the string "D". You could just use `rootNode2` instead.

Reply:
Fixed